hadoop-ozone/integration-test-recon/src/test/java/org/apache/hadoop/ozone/recon/TestReconTasks.java

```diff
@@ -247,7 +248,7 @@ public void testEmptyMissingContainerDownNode() throws Exception {
               .getUnhealthyContainers(
                   ContainerSchemaDefinition.UnHealthyContainerStates.
                       EMPTY_MISSING,
-                  0, 1000);
+                  0L, Optional.empty(), 1000);
```

Comment:
These same changes appear in https://github.com/apache/ozone/pull/8796 , may need rebase of code ?